controllers/suite_test.go

```diff
@@ -141,7 +141,7 @@ var _ = BeforeSuite(func() {
 		RemoteClientCache: remoteClientCache,
 		KcpRestConfig:     k8sManager.GetConfig(),
 	}).SetupWithManager(k8sManager, controller.Options{},
-		controllers.SetupUpSetting{ListenerAddr: UseRandomPort})
+		controllers.SetupUpSetting{ListenerAddr: ":8080"})
```

Comment:
why changed to static port?

Reply:
I reverted it as it is working now with the random port.